### ISSUE TITLE
depth outliers, NMNH Vertebrate Zoology Fishes Collections

### ISSUE THREAD

**Issue by wardappeltans** [iobis/obis-issues#67]
NMNH Vertebrate Zoology Fishes Collections
These are obvious errors (see image), which should be corrected when the dataset moves to OBIS USA @albenson-usgs:

depth | resource_id | id | scientificName
25878 | 268 | 75477 | Diaphus ostenfeldi
25878 | 268 | 1070200 | Hygophum hanseni
25878 | 268 | 3781039 | Nansenia
25878 | 268 | 11339492 | Protomyctophum subparallelum
25878 | 268 | 11500966 | Electrona carlsbergi

![screen shot 2017-08-30 at 17 10 43](https://user-images.githubusercontent.com/15197599/29879719-34701aba-8da6-11e7-8e4a-a3b57ecbd82f.png)

**Comment by albenson-usgs** [comment]
Yes, there are several issues with this particular dataset. We probably need to start fresh with Smithsonian on it.
